buildbot-host/buildmaster/master.cfg

```diff
@@ -655,6 +674,24 @@ factories.update(
 )
 del factory
 
+# MbedTLS openvpn3-linux smoketest builds
```

Comment:
I'm not sure this makes sense any more.  As of `v22_dev`, mbed TLS support has essentially not been accounted for.

Reply:
Are you saying we should disable mbed TLS build altogether or?

Reply:
For openvpn3-linux and openvpn3 there is no one maintaining the mbedTLS port. So if there are any problems with it we just disable the offending build, unless it is trivial to fix. In the case of openvpn3-linux I think there are no other known mbedTLS builds left. For openvpn3 we still have some internally.

Reply:
So yes, I think disabling mbedTLS builds for openvpn3-linux makes sense until someone actually offers to maintain that "port".

Reply:
Yes, I'm pretty sure of dropping mbed TLS support in OpenVPN 3 Linux. It's not even configurable at the moment with the Meson setup in use now.

I'm going to remove the mention of mbed TLS in the BUILD.md as well for the `v23` release.